### PR TITLE
Clarified the A/B options for Swedac

### PR DESCRIPTION
**Pull request by sofiahaglund**
We'll see if the QA approves, I'll leave it as a WIP for now...

### CODE REVIEW

**Comment by sofiahaglund** [comment]
Helena has approved now, so should be ready for code review!

File: data/report_templates/project_summary.md [hunk 1]
```diff
@@ -173,10 +173,10 @@ The naming of the files follow the convention:
 [NGI-NAME]_[BCL-CONVERSION-ID]_[LANE]_[READ]_[VOLUME].fastq.gz
 ```
 
-* _NGI-NAME:_ Internal NGI sample indentifier
-* _BCL-CONVERSION-ID:_ Indentifier set by bcl2fastq tool while demultiplexing
+* _NGI-NAME:_ Internal NGI sample identifier
+* _BCL-CONVERSION-ID:_ identifier set by bcl2fastq tool while demultiplexing
```
Comment (aanil) [inline data/report_templates/project_summary.md:177]: Capitalise I in identifier like in the other lines.

**Review by aanil — APPROVED** [review]
👍

File: data/report_templates/project_summary.md [hunk 2]
```diff
@@ -173,10 +173,10 @@ The naming of the files follow the convention:
 [NGI-NAME]_[BCL-CONVERSION-ID]_[LANE]_[READ]_[VOLUME].fastq.gz
 ```
 
-* _NGI-NAME:_ Internal NGI sample indentifier
-* _BCL-CONVERSION-ID:_ Indentifier set by bcl2fastq tool while demultiplexing
+* _NGI-NAME:_ Internal NGI sample identifier
```
[the remark this answers is not in the document]
Comment (sofiahaglund) [reply data/report_templates/project_summary.md:176]: Same! 🤦‍♀️

File: data/report_templates/project_summary.md (hunk 1 above)
Comment (sofiahaglund) [reply data/report_templates/project_summary.md:177]: Ooooops that's what happens when you use find and replace 😊

**Pull request merged**